Styles/Login.css

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1rem;
+  line-height: 1.3rem;
+}
+
+body {
+  font-family: "Montserrat", sans-serif;
+}
+
+.login-container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  background-color: var(--white-color);
+  transform: translate(-50%, -50%);
+  width: 410px;
+  height: 450px;
+  padding: 30px;
+  box-shadow: 1px 1px 10px var(--black-color);
```

Comment:
px values ? any specific reason you have used it? 

use rems instead. If possible!!